test/scapy/layers/dhcp.uts

```diff
@@ -10,7 +10,9 @@ assert BOOTP().hashret() == b"\x00\x00\x00\x00"
 
 import random
 random.seed(0x2809)
-assert str(RandDHCPOptions(size=1)) in [r"[('NIS_server', '0.45.231.69')]", r"[('tcp_ttl', 229)]"]
+o = str(RandDHCPOptions(size=1))
+# print("RandDHCPOptions %s" % o)
```

Comment:
Could you remove this print?

Reply:
I would propose to leave it. It is pretty useful when the list of options is changing. Currently I needed to figure out these values for MacOS and Windows using your CI on GitHub (I do not have these systems). I also observe that previous PR that touched this also had the same problem.